eclipse/tests/org.switchyard.tools.ui.tests/settings.xml

```diff
@@ -11,7 +11,8 @@
         <repository>
           <id>jboss-public-repository-group</id>
           <name>JBoss Public Maven Repository Group</name>
-          <url>https://repository.jboss.org/nexus/content/groups/public/</url>
+          <url>https://origin-repository.jboss.org/nexus/content/groups/public/</url>
+<!--           <url>https://repository.jboss.org/nexus/content/groups/public/</url> -->
```

Comment:
i would recommend either to remove the commented line or to explain why it is commented